server/issuers.go

```diff
@@ -160,6 +190,95 @@ func (c *Server) issuerGetAllHandler(w http.ResponseWriter, r *http.Request) *ha
 	return nil
 }
 
+// issuerV3CreateHandler - creation of a time aware issuer
+func (c *Server) issuerV3CreateHandler(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+	log := lg.Log(r.Context())
+
+	decoder := json.NewDecoder(http.MaxBytesReader(w, r.Body, maxRequestSize))
+	var req issuerV3CreateRequest
+	if err := decoder.Decode(&req); err != nil {
+		c.Logger.Error("Could not parse the request body")
+		return handlers.WrapError(err, "Could not parse the request body", 400)
+	}
+
+	if req.ExpiresAt != nil {
+		if req.ExpiresAt.Before(time.Now()) {
+			c.Logger.Error("Expiration time has past")
+			return &handlers.AppError{
+				Message: "Expiration time has past",
+				Code:    400,
+			}
+		}
+	} else {
+		// default ExpiresAt
+		req.ExpiresAt = new(time.Time)
+	}
+
+	if err := c.createV3Issuer(Issuer{
```

Comment:
we are not setting the issuer version so its taking the default 0

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/145/commits/2095b05667b6175637a40aead6ea3dbde57e4a01